doc/classes/AtlasTexture.xml

```diff
@@ -1,26 +1,27 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AtlasTexture" inherits="Texture2D" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Crops out one part of a texture, such as a texture from a texture atlas.
+		A texture that crops out part of another Texture2D.
 	</brief_description>
 	<description>
-		[Texture2D] resource that crops out one part of the [member atlas] texture, defined by [member region]. The main use case is cropping out textures from a texture atlas, which is a big texture file that packs multiple smaller textures. Consists of a [Texture2D] for the [member atlas], a [member region] that defines the area of [member atlas] to use, and a [member margin] that defines the border width.
-		[AtlasTexture] cannot be used in an [AnimatedTexture], cannot be tiled in nodes such as [TextureRect], and does not work properly if used inside of other [AtlasTexture] resources. Multiple [AtlasTexture] resources can be used to crop multiple textures from the atlas. Using a texture atlas helps to optimize video memory costs and render calls compared to using multiple small files.
+		[Texture2D] resource that draws only part of its [member atlas] texture, as defined by the [member region]. An additional [member margin] can also be set, which is useful for small adjustments.
+		Multiple [AtlasTexture] resources can be cropped from the same [member atlas]. Packing many smaller textures into a singular large texture helps to optimize video memory costs and render calls.
+		[b]Note:[/b] [AtlasTexture] cannot be used in an [AnimatedTexture], and does not work properly if used inside of other [AtlasTexture] resources.
 	</description>
 	<tutorials>
 	</tutorials>
 	<members>
 		<member name="atlas" type="Texture2D" setter="set_atlas" getter="get_atlas">
-			The texture that contains the atlas. Can be any [Texture2D] subtype.
+			The texture that contains the atlas. Can be any type inheriting from [Texture2D]. It does not work properly with another [AtlasTexture].
```

Comment:
> It does not work properly with another [AtlasTexture]

Is this a bug or by design? If it's by design I think we should say why it will not work - if it is a bug I think we should not mention that here.

Reply:
```suggestion
			The texture that contains the atlas. Can be any type inheriting from [Texture2D]. Nesting [AtlasTexture] is not supported.
```
might be better wording, as "does not work properly" suggests that there is an issue somewhere.